pwiz_tools/Skyline/SkylineTester/TabBuild.cs

```diff
@@ -196,6 +196,8 @@ public static bool CreateBuildCommands(
 
                 commandShell.Add("#@ Checking out {0} source files...\n", branchName);
                 commandShell.Add("# Checking out {0} source files...", branchName);
+                commandShell.Add("# Setting git's http postBuffer size to 500MB, should help with clone on unstable networks");
+                commandShell.Add("{0} config --global http.postBuffer 524288000", git.Quote());

```

Comment:
Doing `git config --global` has the effect of modifying the .gitconfig file in the user's home directory.

I would recommend instead adding `-c http.postBuffer=524288000` to the actual clone commands:
`git -c http.postBuffer=524288000 clone ...`

Reply:
Nice, thanks.